lib/logstash/inputs/twitter.rb

```diff
@@ -121,4 +121,32 @@ def run(queue)
       retry
     end
   end # def run
+
+  protected
+  def process_a_tweet( tweet, queue)
+    if tweet.is_a?(Twitter::Tweet)
```

Comment:
Per http://www.rubydoc.info/gems/twitter/Twitter/Streaming/Client#filter-instance_method:

Yields:

    (Twitter::Tweet, Twitter::Streaming::Event, Twitter::DirectMessage, Twitter::Streaming::FriendList, Twitter::Streaming::DeletedTweet, Twitter::Streaming::StallWarning) —

    A stream of Twitter objects.